com.parallax.PropellerIDE.json

```diff
@@ -0,0 +1,170 @@
+{
+    "app-id": "com.parallax.PropellerIDE",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15",
+    "sdk": "org.kde.Sdk",
+    "command": "propelleride",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--filesystem=host",
+        "--device=all"
+    ],
+    "modules": [
+        {
+            "name": "propelleride",
+            "buildsystem": "qmake",
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/4c0n/PropellerIDE.git",
```

Comment:
If you mean that you'd rather see the original repo here (https://github.com/parallaxinc/PropellerIDE), I've been trying to get this merged for a while now (https://github.com/parallaxinc/PropellerIDE/pull/60) However, the maintainer is a bit difficult to reach. My intentions are to get the patches merged and to update the repo url as soon as they are.
This way I can get everything ready while waiting, instead of just sitting around or making (app) platform specific patches.

Reply:
That's fair, I guess. Can you change it to use specific commit?